### PR TITLE
Add instructions for DAV clients

### LINKED ISSUE #170: Simple instructions to set up calendars & co.
We should provide data to users to set up calendars & co. while having a real calendars & co. administration. It will avoid confusion.

### PR DESCRIPTION
**Pull request by Hywan**
Fix #170.

Show instructions for DAV clients to use Katana.

Here are the screenshots.

Button, to request instructions:
screen shot 2015-05-12 at 17 15 40

Calendars instructions opened:
screen shot 2015-05-12 at 17 15 43

Address books instructions opened:
screen shot 2015-05-12 at 17 15 53

Task lists instructions opened:
screen shot 2015-05-12 at 17 16 03

### CODE REVIEW

**Comment by Hywan** [comment]
Asking a review from @evert, @DominikTo and @demkani.
I will merge this PR and then update the instructions as soon as fruux's instructions will be public.

File: views/admin.html [hunk 1]
```diff
@@ -340,6 +340,27 @@ <h2 class="ui header">Log in</h2>
       <p>No calendar yet.</p>
     {{/each}}
   {{/calendar-list}}
+
+  {{#if showInstructions}}
+    <div class="ui info message">
+      <i {{action 'cancelInstructions'}} class="close icon"></i>
+      <div class="header">Client instructions</div>
+      <p>In the client application, register a new <strong>CalDAV
```
Comment (DominikTo) [inline views/admin.html]: s/register/create/

File: views/admin.html [hunk 2]
```diff
@@ -340,6 +340,27 @@ <h2 class="ui header">Log in</h2>
...
+      account</strong> or a new <strong>calendar</strong> with the following
+      information in manual mode:</p>
+      <ul>
+        <li>Username: <code>{{currentUser}}</code>,</li>
+        <li>Password: <em>user's password</em>,</li>
+        <li>Server address: <code>{{ENV.katana.origin_url}}{{ENV.katana.base_url}}</code>.</li>
+      </ul>
+      <p>Ensure that the “Calendars” option is selected.</p>
```
Comment (DominikTo) [inline views/admin.html]: s/Ensure/Make sure/

File: views/admin.html [hunk 3]
```diff
@@ -357,6 +378,26 @@ <h2 class="ui header">Log in</h2>
       <p>No address book yet.</p>
     {{/each}}
   {{/address-book-list}}
+
+  {{#if showInstructions}}
+    <div class="ui info message">
+      <i {{action 'cancelInstructions'}} class="close icon"></i>
+      <div class="header">Client instructions</div>
+      <p>In the client application, register a new <strong>CardDAV
```
Comment (DominikTo) [inline views/admin.html]: s/register/create/

File: views/admin.html [hunk 4]
```diff
@@ -375,6 +416,27 @@ <h2 class="ui header">Log in</h2>
       <p>No task list yet.</p>
     {{/each}}
   {{/calendar-list}}
+
+  {{#if showInstructions}}
+    <div class="ui info message">
+      <i {{action 'cancelInstructions'}} class="close icon"></i>
+      <div class="header">Client instructions</div>
+      <p>In the client application, register a new <strong>CalDAV
```
Comment (DominikTo) [inline views/admin.html]: s/register/create/

File: views/admin.html [hunk 5]
```diff
@@ -375,6 +416,27 @@ <h2 class="ui header">Log in</h2>
...
+      account</strong> or a new <strong>task list</strong> with the following
+      information in manual mode:</p>
+      <ul>
+        <li>Username: <code>{{currentUser}}</code>,</li>
+        <li>Password: <em>user's password</em>,</li>
+        <li>Server address: <code>{{ENV.katana.origin_url}}{{ENV.katana.base_url}}</code>.</li>
+      </ul>
+      <p>Ensure that the “Task lists” or “Reminders” option is selected.</p>
```
Comment (DominikTo) [inline views/admin.html]: s/Ensure/Make sure/

**Comment by DominikTo** [comment]
Looks good to me (as a first version) besides the few comments :+1:

**Pull request merged**